compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -2811,10 +2811,6 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
         if isErased then
           throw new Exception("Erased function classes are not supported. Use a refined `scala.runtime.ErasedFunction`")
```

Comment:
```suggestion
          throw new Exception("Erased function classes are not supported anymore.")
```

They should not be using this in `3.3` at all.